package.json

```
@@ -61,7 +61,7 @@
     "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
-    "jest": "^24.9.0",
+    "jest": "25",
```

```suggestion
    "jest": "^25.1.0",
```

(and update yarn.lock)

which would be done by using `yarn add --dev jest@latest` instead of `yarn add --dev jest@25`